src/js/components/ToggleButtonGroup/stories/Icons.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Box, ToggleButtonGroup } from 'grommet';
+import { List, MapLocation, Table } from 'grommet-icons';
+
+const optionsIcons = [
+  {
+    icon: <List />,
```

Comment:
We should ensure our examples are demonstrating A11y best practices.

```suggestion
    icon: <List a11yTitle="List view" />,
```

Reply:
Done.